### PR TITLE
Add missing alt text 8

### PR DESCRIPTION
**Pull request by MeeraDi**

### CODE REVIEW

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/install-joomla-on-iis.md [hunk 1]
```diff
@@ -94,29 +94,29 @@ Before starting the installation procedure for Joomla, you must create a databas
...
-    [![](install-joomla-on-iis/_static/image10.jpg)](install-joomla-on-iis/_static/image9.jpg)
+    ![Screenshot of the Joomla installation page. F T P configuration settings are showing in the main pane.](install-joomla-on-iis/_static/image9.jpg)
 
     *Figure 5: FTP Configuration page*
 7. Specify the *site name* and the *administrator password*. Click **Next** to complete the installation.  
 
     Note: You may scroll down and install sample data if you prefer.
 
-    [![](install-joomla-on-iis/_static/image12.jpg)](install-joomla-on-iis/_static/image11.jpg)
+    ![Screenshot of the Joomla installation  page. Main configuration settings are showing in the main pane.](install-joomla-on-iis/_static/image11.jpg)
```
Comment (TimShererWithAquent) [inline iis/application-frameworks/install-and-configure-php-applications-on-iis/install-joomla-on-iis.md:109]: extra space before "page"

File: iis/develop/using-webmatrix/download-and-install-a-php-application.md [hunk 2]
```diff
@@ -32,30 +32,30 @@ The following items are required to complete the procedures in this walkthrough:
...
 > [!NOTE]
 > The root account is used for administering an instance of the MySQL database engine.
 
-[![](download-and-install-a-php-application/_static/image15.png)](download-and-install-a-php-application/_static/image13.png)
+![Screenshot of the page for entering the database administrator password.](download-and-install-a-php-application/_static/image13.png)
 
 On the **Accept EULA** page, read the terms, and if you accept them, click **I Accept**.
 
-[![](download-and-install-a-php-application/_static/image19.png)](download-and-install-a-php-application/_static/image17.png)
+![Screenshot of the Accept EULA page.](download-and-install-a-php-application/_static/image17.png)
```
Comment (TimShererWithAquent) [inline iis/develop/using-webmatrix/download-and-install-a-php-application.md:54]: Also under 40.

File: iis/develop/using-webmatrix/download-and-install-a-php-application.md [hunk 3]
```diff
@@ -66,18 +66,18 @@ When you are prompted, enter the following information:
...
 
 When the installation has completed, WebMatrix lists everything that was installed. Click **OK** to close the **Site from Web Gallery** wizard.
 
-[![](download-and-install-a-php-application/_static/image35.png)](download-and-install-a-php-application/_static/image33.png)
+![Screenshot showing a list of successfully installed products.](download-and-install-a-php-application/_static/image33.png)
 
 When the **Site from Web Gallery** wizard has closed, WebMatrix automatically opens the WordPress application.
 
-[![](download-and-install-a-php-application/_static/image39.png)](download-and-install-a-php-application/_static/image37.png)
+![Screenshot of the WordPress page.](download-and-install-a-php-application/_static/image37.png)
```
Comment (TimShererWithAquent) [inline iis/develop/using-webmatrix/download-and-install-a-php-application.md:81]: And under 40.

File: iis/get-started/whats-new-in-iis-8/iis-80-cpu-throttling-sand-boxing-sites-and-applications.md [hunk 4]
```diff
@@ -56,32 +56,32 @@ There are no known bugs for this feature at this time.
 
 1. Open **IIS Manager**.
 2. Select **Application Pools** in the left navigation window:  
-    ![](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image1.png)
+    ![Screenshot showing the connections pane with Application Pools highlighted.](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image1.png)
```
Comment (TimShererWithAquent) [inline iis/get-started/whats-new-in-iis-8/iis-80-cpu-throttling-sand-boxing-sites-and-applications.md:59]: Should "connections" be capitalized?

File: iis/get-started/whats-new-in-iis-8/iis-80-cpu-throttling-sand-boxing-sites-and-applications.md [hunk 5]
```diff
@@ -56,32 +56,32 @@ There are no known bugs for this feature at this time.
...
 3. Select **DefaultAppPool**:  
-    ![](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image3.png)
+    ![Screenshot showing the Applications Pools page with DefaultAppTool highlighted.](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image3.png)
 4. In the **Action** pane, select **Advanced Settings**:  
-    ![](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image5.png)
+    ![Screenshot of the Actions pane with Advanced Settings highlighted.](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image5.png)
 5. Under **CPU** group, locate the following configurations:  
-    ![](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image7.png)
+    ![Screenshot of the Advanced Settings dialog. The CPU section is highlighted.](iis-80-cpu-throttling-sand-boxing-sites-and-applications/_static/image7.png)
```
Comment (TimShererWithAquent) [inline iis/get-started/whats-new-in-iis-8/iis-80-cpu-throttling-sand-boxing-sites-and-applications.md:65]: CPU > C P U

File: iis/manage/configuring-security/using-dynamic-ip-restrictions.md [hunk 6]
```diff
@@ -58,11 +58,11 @@ To access Dynamic IP Restriction settings in IIS Manager follow these steps:
 1. Open IIS Manager
 2. In the left-hand side tree view select server node if you want to configure server-wide settings, or select a site node to configure site-specific settings.
 3. In the Features View click "Dynamic IP Restrictions"  
-    ![](using-dynamic-ip-restrictions/_static/image1.jpg)
+    ![Screenshot of I I S Manager showing the features list in the main pane, Dynamic IP Restrictions is selected.](using-dynamic-ip-restrictions/_static/image1.jpg)
```
Comment (TimShererWithAquent) [inline iis/manage/configuring-security/using-dynamic-ip-restrictions.md:61]: IP > I P

**Comment by MeeraDi** [comment]
@Rick-Anderson, @mcleblanc, @v-andreaco
This PR addresses missing alt text and is ready for final review.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**